### PR TITLE
Update Matterbridge docs

### PR DESCRIPTION
**Pull request by epassaro**
## Description
A small update to Matterbridge section in docs.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have assigned/requested two reviewers for this pull request.

### CODE REVIEW

File: docs/development/matterbridge.rst [hunk 1]
```diff
@@ -1,52 +1,61 @@
...
+later in the TOML file.
 
 
 ------
 Gitter
 ------
 
 Create a new GitHub dedicated account to use as a bot, refer to `this link`_
-and copy the token.
+and copy the token o use later in the TOML file.
```
Comment (smithis7) [inline docs/development/matterbridge.rst:51]:
```suggestion
and copy the token to use later in the TOML file.
```

Just a small typo-- I'm not sure if that's what I should be looking for or not though.

**Review by smithis7 — CHANGES_REQUESTED** [review]

File: docs/development/matterbridge.rst (hunk 1 above)
Comment (epassaro) [reply docs/development/matterbridge.rst:51]: Yes, it is. Thanks! :smile:

**Review by smithis7 — APPROVED** [review]

**Review by smithis7 — APPROVED** [review]

File: docs/development/matterbridge.rst [hunk 2]
```diff
@@ -1,52 +1,61 @@
...
+messaging platforms as you want, like *tokens* and *passwords*. Once the
+application is running, messages start be shared between the connected rooms.
 
-Currently, we keep a service running on `OpenSupernova.org`_ server to keep
-running Matterbridge as a daemon. Our configuration files are stored in the
-private repository ``tardis-matterbridge``. These files include our custom
-TOML configuration file and the files needed to set up the Linux service.
+Currently, we keep a service running on `OpenSupernova.org`_ server to 
+run Matterbridge as a daemon. Our configuration files are stored in a private 
+GitHub repository, includin our custom TOML and the files needed to set up the
```
Comment (andrewfullard) [inline docs/development/matterbridge.rst:21]:
```suggestion
GitHub repository, including our custom TOML and the files needed to set up the
```

File: docs/development/matterbridge.rst [hunk 3]
```diff
@@ -1,52 +1,61 @@
...
+such as *Slack*, *Mattermost* and *Gitter*. Matterbridge provides binaries 
+for  many operating systems. The ``matterbridge`` binary is intended to be used
+in combination with a TOML configuration file::
   $ ./matterbridge -conf config-tardis-matterbridge.toml
 
-The TOML file includes all the configuration needed to connect as many
-messaging platforms as you want, including tokens and passwords.
+The TOML file includes all the parameters required to connect as many
+messaging platforms as you want, like *tokens* and *passwords*. Once the
+application is running, messages start be shared between the connected rooms.
```
Comment (andrewfullard) [inline docs/development/matterbridge.rst:17]:
```suggestion
application is running, messages can be shared between the connected rooms.
```

File: docs/development/matterbridge.rst [hunk 4]
```diff
@@ -1,52 +1,61 @@
...
 
-Currently, we keep a service running on `OpenSupernova.org`_ server to keep
-running Matterbridge as a daemon. Our configuration files are stored in the
-private repository ``tardis-matterbridge``. These files include our custom
-TOML configuration file and the files needed to set up the Linux service.
+Currently, we keep a service running on `OpenSupernova.org`_ server to
```
Comment (andrewfullard) [inline docs/development/matterbridge.rst:19]:
```suggestion
Currently, we keep a service running on the `OpenSupernova.org`_ server to 
```

File: docs/development/matterbridge.rst [hunk 5]
```diff
@@ -1,52 +1,61 @@
...
+A *cron* job restarts the service periodically to prevent disconnections.
 
 
 =====
 Setup
 =====
 
 Use the TOML file in ``tardis-matterbridge`` as an example. You can set up
-as many gateways as you want.
+as many gateways as you want!.
```
Comment (andrewfullard) [inline docs/development/matterbridge.rst:34]:
```suggestion
as many gateways as you want!
```

**Review by andrewfullard — CHANGES_REQUESTED** [review]

**Review by andrewfullard — APPROVED** [review]

**Pull request merged**